v2/support/instances_sample_app/README.md

```diff
@@ -0,0 +1,8 @@
+:warning: *Use of this software is subject to important terms and conditions as set forth in the License file* :warning:
+
+# Instances API Sample App
+
+This sample app uses the [Instances API](https://developer.zendesk.com/apps/docs/apps-v2/api_reference#client.instanceguid) to add a template comment from the background location when the comment box is empty.
```

Comment:
Worth mentioning/explaining the autoLoad: false property for the ticket sidebar used here?